src/hash.cpp

```diff
@@ -249,3 +250,7 @@ uint64_t SipHashUint256Extra(uint64_t k0, uint64_t k1, const uint256& val, uint3
     SIPROUND;
     return v0 ^ v1 ^ v2 ^ v3;
 }
+
+SaltedHasherBase::SaltedHasherBase() : k0(GetRand(std::numeric_limits<uint64_t>::max())), k1(GetRand(std::numeric_limits<uint64_t>::max())) {}
```

Comment:
This thing doesn't belong here it seems, should be moved out of `consensus` stuff and into `util` or `common` somewhere.

Reply:
The salted hashers got their own source files now to avoid this problem.

Reply:
👍
Let's see if Travis is happy this time :)